doc-surrealdb_versioned_docs/version-latest/querying/surrealql/http.mdx

```diff
@@ -0,0 +1,163 @@
+            </td>
+        </tr>
+        <tr>
+            <td colspan="2" scope="row" data-label="Header">
+                <code>NS</code>
+                <l className='yellow'>REQUIRED</l>
+                 <l className='purple'>V1.x</l>
+            </td>
+            <td colspan="2" scope="row" data-label="Description">
+                Sets the selected Namespace for queries
+            </td>
+        </tr>
+        <tr>
+            <td colspan="2" scope="row" data-label="Header">
+                <code>DB</code>
+                <l className='yellow'>REQUIRED</l>
+                <l className='purple'>V1.x</l>
+            </td>
+            <td colspan="2" scope="row" data-label="Description">
+                Sets the selected Database for queries
+            </td>
+        </tr>
+        <tr>
+            <td colspan="2" scope="row" data-label="Header">
+                <code>surreal-ns</code>
+                <l className='yellow'>REQUIRED</l>
+                 <l className='purple'>V2.x</l>
+            </td>
+            <td colspan="2" scope="row" data-label="Description">
+                Sets the selected Namespace for queries
+            </td>
+        </tr>
+        <tr>
+            <td colspan="2" scope="row" data-label="Header">
+                <code>surreal-db</code>
+                <l className='yellow'>REQUIRED</l>
+                <l className='purple'>V2.x</l>
+            </td>
+            <td colspan="2" scope="row" data-label="Description">
+                Sets the selected Database for queries
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### Example usage
+```bash title="Request"
+curl -X POST -u "root:root" -H "NS: mynamespace" -H "DB: mydatabase" -H "Accept: application/json" -d "SELECT * FROM person WHERE age > 18" http://localhost:8000/sql
+```
+
+<Tabs groupId="http-sql">
+  <TabItem value="V1" label="V1.x" default>
+
+```bash title="Request"
+curl -X POST -u "root:root" -H "NS: mynamespace" -H "DB: mydatabase" -H "Accept: application/json" -d "SELECT * FROM person WHERE age > 18" http://localhost:8000/sql
+```
+
+</TabItem>
+<TabItem value="CLI" label="V2.x">
+```bash title="Request"
+curl -X POST -u "root:root" -H "surreal-ns: mynamespace" -H "surreal-db: mydatabase" -H "Accept: application/json" -d "SELECT * FROM person WHERE age > 18" http://localhost:8000/sql
+```
+</TabItem>
+</Tabs>
+
+
+
+```json title="Response"
+[
+	{
+		"time": "14.357166ms",
+		"status": "OK",
+		"result": [
+			{
+				"age": "23",
+				"id": "person:6r7wif0uufrp22h0jr0o"
+				"name": "Simon",
+			},
+			{
+				"age": "28",
+				"id": "person:6r7wif0uufrp22h0jr0o"
+				"name": "Marcus",
+			},
+		]
+	}
+]
+```
+
+## Using Postman
+
+Postman is a popular tool for testing APIs. You can use it to send HTTP requests to your SurrealDB instance and perform various database operations.
+
+1. **Set up Postman**: Download and install Postman from the [official website](https://www.postman.com/).
```

Comment:
Maybe ignore my comment above about Postman since I see the download and usage instructions are all here.